Blog contributions (#1581)
o3de/o3de.org: content/docs/contributing/to-docs/blog-posts.md

To draft and submit a blog post, you need:

* A GitHub account known to O3DE and a basic understanding of simple Git workflows
* A working knowledge of Markdown syntax

If you have never participated in our communities before, we highly recommend you spend some time socializing in them and joining at least one SIG meeting before you start blogging. Regardless, blogs will be reviewed by community maintainers before they are published.

## How the O3DE blog works

Blog posts are submitted through the same process as the [docs contribution workflow](./get-started).

1. Fork the https://github.com/o3de/o3de.org repo.
1. Clone your fork to your local machine. Set `upstream` to o3de.org and `origin` to your fork.
1. Create a branch for submission. This branch should contain _only_ the necessary content additions for your blog post.

> 1. Open your editor of choice and create a draft post using Markdown under thr `/content/blog/posts` folder. Give it a clear, unique filename in all lowercase, with no spaces. (Use a hyphen `-` as a replacement for spaces. Please do not use any other non-alphanumeric characters.)

Reviewer: ```suggestion
1. Open your editor of choice and create a draft post using Markdown under the `/content/blog/posts` folder. Give it a clear, unique filename in all lowercase, with no spaces. (Use a hyphen `-` as a replacement for spaces. Please do not use any other non-alphanumeric characters.)
```

1. Make sure that your blog post has the following metadata at the top:

    ```metadata
    ---
    title: The title of your blog post. Please keep it under 80 characters.
    date: YYYY-MM-DD
    slug: <Unique string for your post, usually a condensed form of the title>
    author: YOUR PREFERRED AUTHOR NAME HERE. EMPLOYER IS OPTIONAL!
    blog_img: "/images/blog/<optional thumbnail>"
    full_img: "/images/blog/<optional full image>"
    ---
    ```

    As an example:

    ```metadata
    ---
    title: Announcing Open 3D Engine!
    date: 2021-07-06
    slug: welcome-post
    author: Doug Erickson, Amazon Web Services
    blog_img: "/images/blog/announcement_thumbnail.jpg"
    full_img: "/images/blog/atom_showcase.png"
    ---
    ```

    {{ note }}

>     Blog posting is automated and that the `date` field determines when the blog will first become visible if your pull request is accepted and merged.

Reviewer: ```suggestion
    Blog posting is automated and the `date` field determines when the blog will first become visible if your pull request is accepted and merged.
```

    {{ /note }}
1. Commit your changes to your branch. **Ensure your commit(s) are DCO signed!**
1. Push your branch to your fork (`origin`). Go to your fork on GitHub and create a pull request from your fork/branch to `o3de/o3de.org` against the `main` branch.
1. Go to your pull request at https://github.com/o3de/o3de.org/pulls and ensure that all checks have passed. If they haven't, fix up your pull request and resubmit it.

> 1. Go into the [O3DE Discord](https://discord.gg/o3de) and share your pull request for review, or send an email to the [O3DE Documentation & Community SIG](https://lists.o3de.org/g/sig-docs-community) mailing list. Someone will review it and provide feedback or an okay. For blog posts, we go through the regular pull request review process and have an additional requirement that a relevant SIG, the TSC, or the O3DE marketing committee approve the blog post.

Reviewer: ```suggestion
1. Go into the [O3DE Discord](https://discord.gg/o3de) and share your pull request for review, or send an email to the [O3DE Documentation & Community SIG](https://lists.o3de.org/g/sig-docs-community) mailing list. Someone will review it and provide feedback or an okay. For blog posts, we go through the regular pull request review process with an additional requirement that a relevant SIG, the TSC, or the O3DE marketing committee approve the blog post.
```

 
{{< note >}}
The Open 3D Foundation and the broader Open 3D Engine community reserve the right to not publish any blog post that does not meet our standards. We also reserve the right to remove any currently published blog post.
{{< /note >}}

## What makes a good O3DE blog post?

Good question! That's really up to the community alongside our established Code of Conduct and Community Tenets. Not sure? Join the O3DE Discord or one of the SIGs, and vet your idea with our members.

> Here's some basic Do's and Don'ts to help you focus your work and avoid rejection.

Reviewer: ```suggestion
Here are some basic "Dos and Don'ts" to help you focus your work and avoid rejection.
```
I know it looks weird, but there isn't an apostrophe in 'dos'.

* Write with strong grammar and an eye for easy readability. Rambling monologues may get a gentle request for editorial review and a number of suggestions to "tighten it up." The reader's time is precious, so compressing your thoughts into short, information-dense sentences with regular paragraph breaks is just respectful.

* Include images and diagrams wherever appropriate.

* Consider your audience, which is diverse across many sociocultural and industry axes. Choose your words with respect and care, and be prepared to take feedback sincerely and make changes. No one knows everything about the folks in the community and the industry, so don't let some initially careless wording caught in review bring you down. Make the changes (as appropriate) and keep delivering great content!

### Don't

* Get personal. Do not use the O3DE blog to call out any individuals in a negative way.

> * Get political. Politics pervade every part of life, and we all have different backgrounds, ideologies, and reactions to the way in which culture and society shapes itself. The O3DE blog isn't the place for it, unless you are running a community-approved political initiative and have the blessings of the Linux Foundation.

Reviewer: ```suggestion
* Get political. Politics pervade every part of life, and we all have different backgrounds, ideologies, and reactions to the way in which society shapes itself. The O3DE blog isn't the place for it&mdash;unless you are running a community-approved political initiative and have the blessings of the Linux Foundation.
```
Removed culture to avoid 'culture and society shape themselves'.  Em dashes are used elsewhere; it's the right choice here instead of a comma.
